app/policies/comment_policy.rb

```diff
@@ -65,8 +65,8 @@ def user_is_moderator?
     user.moderator_for_tags.present?
   end
 
-  def user_is_comment_banned?
-    user.has_role? :comment_banned
+  def user_is_comment_suspended?
+    user.has_role? :comment_suspended
```

Comment:
Let's be a bit more consistent about the style we use:

```suggestion
    user.has_role?(:comment_suspended)
```